specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -11829,6 +11861,10 @@
           "description": "The selected recovery azure network Id.",
           "type": "string"
         },
+        "selectedTfoAzureNetworkId": {
```

Comment:
>AzureNetworkId [](start = 20, length = 14)

why do these refer to "AzureNetworkId" instead of "VirtualNetworkId"?

Reply:
As we use virtual network id for on-premise machines, we refer AzureNetworkId for Azure failed over vms. Tfo is the acronym familiar to ASR users for Test Failover.